packages/react-app/pages/profile/approval/index.js

```diff
@@ -33,6 +34,27 @@ function ApproveShareContactInformation() {
       setMyPrivateProfile(privateProfile);
       const reqs = await dRecruitV1Contract.getRequesters(privateProfile.tokenId);
       setRequesters(reqs);
+      console.log(context.injectedProvider);
+      const core = ceramicCoreFactory();
+      const recruiterDIDs = await Promise.all(
+        reqs.map(recruiterAddress =>
+          core.getAccountDID(`${recruiterAddress}@eip155:${context.injectedProvider._network.chainId}`),
```

Comment:
I think that you can use `context.targetNetwork.chainId` instead, otherwhise we might run into issues on localhost.